src/Datadog.Trace.ClrProfiler.Managed/Integrations/MongoDbIntegration.cs

```diff
@@ -50,7 +51,7 @@ public static object Execute(object wireProtocol, object connection, object canc
                 }
                 catch (Exception ex)
                 {
-                    scope.Span.SetException(ex);
+                    scope?.Span.SetException(ex);
```

Comment:
I realize this integration didn't change a lot, but there's a lot of the same issues here as others (potential leak in the using/create scope, use an exception filter vs. catching) that should be addressed, and others that probably should wait (i.e. moving to external dependencies)